aspnetcore/fundamentals/error-handling.md

```diff
@@ -2,65 +2,45 @@
+Place the call to <xref:Microsoft.AspNetCore.Builder.DeveloperExceptionPageExtensions.UseDeveloperExceptionPage*> in front of any middleware where you want to catch exceptions.
 
 > [!WARNING]
-> Enable the Developer Exception Page **only when the app is running in the Development environment**. You don't want to share detailed exception information publicly when the app runs in production. [Learn more about configuring environments](xref:fundamentals/environments).
-
-To see the Developer Exception Page, run the sample app with the environment set to `Development` and add `?throw=true` to the base URL of the app. The page includes several tabs with information about the exception and the request. The first tab includes a stack trace:
-
-![Stack trace](error-handling/_static/developer-exception-page.png)
-
-The next tab shows the query string parameters, if any:
-
-![Query string parameters](error-handling/_static/developer-exception-page-query.png)
+> Enable the Developer Exception Page **only when the app is running in the Development environment**. You don't want to share detailed exception information publicly when the app runs in production. For more information on configuring environments, see <xref:fundamentals/environments>.
 
-If the request has cookies, they appear on the **Cookies** tab. Headers are seen in the last tab:
+To see the Developer Exception Page, run the sample app with the environment set to `Development` and add `?throw=true` to the base URL of the app. The page includes the following information about the exception and the request:
 
-![Headers](error-handling/_static/developer-exception-page-headers.png)
+* Stack trace
+* Query string parameters (if any)
+* Cookies (if any)
+* Headers
 
 ## Configure a custom exception handling page
 
 Configure an exception handler page to use when the app isn't running in the `Development` environment:
 
-[!code-csharp[](error-handling/samples/2.x/ErrorHandlingSample/Startup.cs?name=snippet_DevExceptionPage&highlight=11)]
+[!code-csharp[](error-handling/samples/2.x/ErrorHandlingSample/Startup.cs?name=snippet_DevExceptionPage&highlight=9)]
 
-In a Razor Pages app, the [dotnet new](/dotnet/core/tools/dotnet-new) Razor Pages template provides an Error page and an error `PageModel` class in the *Pages* folder.
+In a Razor Pages app, the [dotnet new](/dotnet/core/tools/dotnet-new) Razor Pages template provides an Error page and an error <xref:Microsoft.AspNetCore.Mvc.RazorPages.PageModel> class in the *Pages* folder.
 
 In an MVC app, don't decorate the error handler action method with HTTP method attributes, such as `HttpGet`. Explicit verbs prevent some requests from reaching the method. Allow anonymous access to the method so that unauthenticated users are able to receive the error view.
 
```

Comment:
In case of following text also, why are we talking only about dotnet new and not VS templates?